src/applications/vaos/tests/new-appointment/components/TypeOfCarePage.unit.spec.jsx

```diff
@@ -417,6 +416,11 @@ describe('VAOS <TypeOfCarePage>', () => {
 
       fireEvent.click(await screen.findByLabelText(/primary care/i));
       fireEvent.click(screen.getByText(/Continue/));
+      await waitFor(() =>
```

Comment:
I think this test will fail already if this URL is wrong or the fetch isn't made, so this assertion might be overkill.